mountpoint-s3/Cargo.toml

```diff
@@ -7,7 +7,7 @@ publish = false
 
 [dependencies]
 fuser = { path = "../vendor/fuser", version = "0.14.0", features = ["abi-7-28"] }
-mountpoint-s3-client = { path = "../mountpoint-s3-client", version = "0.6.2" }
+mountpoint-s3-client = { path = "../mountpoint-s3-client", version = "0.6.2", default-features = false }
```

Comment:
Is `default-features=false` required? Or just for extra caution?

Reply:
Nope, just a leftover from when I had mock enabled by default at some point in writing this. Fixed.